examples/xonotic/Makefile

```diff
@@ -39,16 +39,20 @@ image_tag = $(REPOSITORY)/xonotic-example:1.1
 
 # Build a docker image for the server, and tag it
 build:
-	docker build -f $(project_path)/Dockerfile --tag=$(image_tag) .
+	docker build -f $(project_path)/Dockerfile --tag=$(REPOSITORY)/$(image_tag) .
 
 # Push a docker image to the repository, and save it
 push: build
-	docker push $(image_tag)
+	docker push $(REPOSITORY)/$(image_tag)
 
 # check if hosted on Google Artifact Registry
 gar-check:
-	gcloud container images describe $(image_tag)
+	gcloud container images describe $(REPOSITORY)/$(image_tag)
```

Comment:
I think we'll need something like a `PRODUCTION_REPOSITORY` variable (or something like that) so that `gar-check` and `echo-image-tag` still works as it did previously (since they are used for the release process).

Reply:
Included PROD_REPO variable with `agones-images` but I'm unsure about the behavior of gar-check and echo-image-tag targets🤔